Gemfile

```diff
@@ -258,6 +258,7 @@ end
 # Reduce volume of production logs
 # Ref: https://github.com/roidrage/lograge/pull/252
 gem 'lograge', github: 'code-dot-org/lograge', ref: 'debug_exceptions'
+gem 'request_store', '~> 1.6.0', require: false
```

Comment:
We need to allow `request_store` in any group where `lograge` (`debug_exceptions`) is enabled, since `lograge` depends on it.

https://github.com/code-dot-org/code-dot-org/blob/6f3f113491d7a65cd148abdabcfda73d9e77c83e/Gemfile#L260

https://github.com/code-dot-org/code-dot-org/blob/6f3f113491d7a65cd148abdabcfda73d9e77c83e/Gemfile.lock#L23-L31